lib/widgets/bubbles.dart

```diff
@@ -200,32 +201,37 @@ class LabelBubble extends StatelessWidget {
 
   LabelBubble(this.title, this.position, this.dimensions, this.homePageState);
 
-  double computeInnerBubbleSize() {
+  /// Returns the size of the inner bubble according to the probability returned by [getLabelProbabilities]
+  /// TODO currently assumes that the label title is the same as the label name, just camel case.
```

Comment:
Do we have access to label titles ?

Reply:
You mean to the label config?

Reply:
Correct. if yes, we should be able to just take them instead of having this TODO

Reply:
Currently the LabelBubbble only knows its label's title. So we woulf need a reversed dictionary to get the actual name. Currently in labels.config we have `age -> Age` but would need `Age -> age` to use it here. That's why the TODO because it's not a too easy fix.

Reply:
if we have all the configs I'm not sure I see the problem, but either way the comment of "label title is same as label name" might as well be rocket science. :man_astronaut:

Reply:
```suggestion
```